test/VisualizationTests/VisualizationDataTests.jl

```diff
@@ -0,0 +1,31 @@
+module TestVisualizationData
+
+using Test
+using Gridap
+using Gridap.Visualization
+
+domain = (0,2pi, 0, 4pi)
+partition = (2, 3)
+model = simplexify(CartesianDiscreteModel(domain,partition))
+f(pt) = sin(pt[1])
+V0 = TestFESpace(
+  reffe=:Lagrangian, order=1, valuetype=Float64,
+  conformity=:H1, model=model)
+U = TrialFESpace(V0)
+trian = Triangulation(model)
+degree = 2
+quad = CellQuadrature(trian,degree)
+A(u, v) = u ⊙ v
+b(v) = v⊙f
+t_Ω = AffineFETerm(A,b,trian,quad)
+op = AffineFEOperator(U,V0,t_Ω)
+u = solve(op)
+
+visdata = visualization_data(trian, cellfields=Dict("u" =>u))
+ncells = 12
+nodes_per_cell = 3
```

Comment:
How to extract these numbers directly from `trian` or `model`?

Reply:
```julia
using Gridap.Geometry
using Gridap.ReferenceFEs
ncells = num_cells(vistrian)
nodes_per_cell = num_nodes(first(get_reffes(vistrian)))
```